### ISSUE TITLE
React 15 support

### ISSUE THREAD

**Issue by axelg12** [brigade/react-waypoint#74]
With the newly released React 15, the peer dependency is outdated and will not install.
```
react-waypoint@2.0.0 requires a peer of react@^0.14.0 but none was installed.
react-waypoint@2.0.0 requires a peer of react-dom@^0.14.0 but none was installed.
```
The quick way to fix this, I think, is to change the line from 
```
"peerDependencies": {
    "react": "^0.14.0",
    "react-dom": "^0.14.0"
  },
```
to 
```
"peerDependencies": {
    "react": "^0.14.0 || ^15.0.0",
    "react-dom": "^0.14.0 || ^15.0.0""
  },
```
This will allow people to use it with both 0.14 and 15.0

**Comment by trotzig** [comment]
That sounds good to me. Would you mind putting that together in a PR? You should bump the devDependency at the same, so that we start developing (and testing) with 15.0, but allow peer dependencies back to 0.14.

**Comment by trotzig** [comment]
Whoops, I didn't realize someone else beat us. :)

**Comment by trotzig** [comment]
Fixed by a6b86453d5.
